portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/ExposeLanguageFromAssetLibraryAndSite.testcase

```diff
@@ -0,0 +1,91 @@
+@component-name = "portal-headless-frontend-infrastructure"
```

Comment:
Could you please rename the file to use languages, to be ExposeLanguagesFromAssetLibraryAndSite.testcase? Thanks